x-pack/solutions/observability/plugins/slo/server/services/create_slo.ts

```diff
@@ -100,14 +100,14 @@ export class CreateSLO {
       ]);
     } catch (err) {
       this.logger.error(
-        `Cannot install the SLO [id: ${slo.id}, revision: ${slo.revision}]. Rolling back.`
+        `Cannot create the SLO [id: ${slo.id}, revision: ${slo.revision}]. Rolling back. ${err}`
```

Comment:
Isn't `err` an object? How would it be logged in this case?
I vaguely remember there was a consideration while logging errors to ensure we don't log sensitive information, do you remember what the issue was and whether it is applicable in this PR?

Reply:
in this case, it would be just formatted as error string.